reference-architecture/gce-cli/ansible/playbooks/roles/openshift-ansible-installer/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: download openshift ansible installer from github
+  git:
+    repo: https://github.com/openshift/openshift-ansible.git
+    dest: '{{ playbook_dir }}/../../../../../openshift-ansible'
```

Comment:
OCP subscription provides 10x subscriptions[1] for infra stuff (including infra nodes, masters, bastions) on every node subscription, so I agree it stinks but it shouldn't be a problem regarding subscriptions.
[1] https://access.redhat.com/solutions/2851701